pages/dkp/kommander/2.1/projects/applications/i-dont-know-where-to-keep-this.md

```diff
@@ -0,0 +1,83 @@
+    ```
+
+<!-- TODO: should we just ask to create a new user? Its less risky, but difficult for users as they need to remember the username and password for future-->
+1. Using the above username and password, login to gitea UI and create a new repository under `kommander` organization. It's also possible to register a new user and then create a repository under that new username. Rest of this document assumes you are using `kommander` organization. You should be able to substitute `kommander` with your own organization name.
+    
+    ```bash
+    kubectl -n kommander get ingress gitea -o go-template='https://{{ (index .status.loadBalancer.ingress 0).hostname }}:443/dkp/kommander/git'
+    ```
+
+1. Login to gitea dashboard with `GITEA_USERNAME` and `GITEA_PASSWORD` credentials. Create a new repository named `dkp-catalog-applications` or any other name you prefer.
+
+1. Clone the newly created repository on your local machine
+
+    ```bash
+    git clone -c http.sslVerify=false https://$GITEA_USERNAME:$GITEA_PASSWORD@$(kubectl -n kommander get ingress gitea -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'):443/dkp/kommander/git/kommander/dkp-catalog-applications
+    ```
+
+1. Download and extract the catalog repository bundle on to your local machine from download portal and extract the contents into `dkp-catalog-applications` cloned earlier.
+
+    ```bash
+    curl -fsSL https://github.com/mesosphere/dkp-catalog-applications/archive/refs/tags/v2.1.1-rc.2.tar.gz | tar zxf - --strip-components=1 -C dkp-catalog-applications
+    ```
+
+1. Navigate into `dkp-catalog-applications` and push the changes.
+
+    ```bash
+    cd dkp-catalog-applications
+    git add .
+    git commit -m "feat: initialize dkp-catalog-applications for airgapped"
+    git push --set-upstream origin master
+    ```
+
+1. Run the following command to create a secret containing Certificate Authority information into `TARGET_NAMESPACE`
+
+    ```bash
+    k create secret generic -n${TARGET_NAMESPACE} ${TARGET_NAMESPACE} --from-literal=caFile="$(kubectl get secrets -nkommander-flux kommander-git-credentials -o template='{{ .data.caFile | base64decode }}')" --type opaque
+    ```
+
+    **NOTE**: This command needs to be run in ALL target clusters that belong to the specific Workspace or Project. 
+    <!-- TODO: Create a federated secret maybe? -->
+
+1. Finally, run the following command to create the catalog `GitRepository`
+
+    <!-- TODO: support custom host name -->
+    ```bash
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${TARGET_NAMESPACE}
+    spec:
+      interval: 1m0ss
+      ref:
+        branch: master
+      timeout: 20s
+      # TODO: use different url so this works in attached clusters
```

Comment:
??? We can't leave this in here.  Is it meant to be a comment in the docs?  If not, how do you do this?

```suggestion
```

Reply:
yep, i still need to work on the `TODO`.